src/Compositor.cpp

```diff
@@ -2531,8 +2532,9 @@ Vector2D CCompositor::parseWindowVectorArgsRelative(const std::string& args, con
     bool        yIsPercent = false;
     bool        isExact    = false;
 
-    std::string x = args.substr(0, args.find_first_of(' '));
-    std::string y = args.substr(args.find_first_of(' ') + 1);
+    CVarList varList(args, 2, ' ', true);
+    std::string x = varList.size() > 0 ? varList[0] : "";
+    std::string y = varList.size() > 1 ? varList[1] : "";
```

Comment:
the point of that fix kinda got fucked, tho. Make it max 3 and in `exact` just use `[1]` and `[2]`

Reply:
My bad. Thanks for the review! I hope that now it is good.

Reply:
and probably `'s'` as sep? or it will fuckup tabs in conf

Reply:
It is working fine with ' ' as delimiter even if I put tabs between args, but I don't if in some case 's' would be preferred here.